Source/SwiftLintFrameworkTests/ASTRuleTests.swift

```diff
@@ -148,6 +148,10 @@ class ASTRuleTests: XCTestCase {
             ruleDescription: FunctionBodyLengthRule.description,
             location: Location(file: nil, line: 1, character: 1),
             reason: "Function body should span 40 lines or less: currently spans 41 lines")])
+        let longerFunctionBodyWithComments = "func abc() {" +
+            Repeat(count: 40, repeatedValue: " // this is a comment\n").joinWithSeparator("") +
+            "}\n"
+        XCTAssertEqual(violations(longerFunctionBodyWithComments), [])
```

Comment:
That tests actually caught a bug: the number of lines that the function spans in the reason. Should we update it to subtract the comment only lines? This might be confusing for someone who isn't very familiar with the rule.

Also, if we follow that idea, any ideas on how to return the "fixed"  line count on `exceedsLineCountExcludingComments`? I was thinking about returning a tuple, but I'm not sure if that's the best way.